plugins/UM3NetworkPrinting/src/UM3OutputDevicePlugin.py

```diff
@@ -215,6 +215,26 @@ def addManualDevice(self, address):
 
         self._checkManualDevice(address)
 
+    def _createMachineFromDiscoveredPrinter(self, key: str) -> None:
+        # TODO: This needs to be implemented. It's supposed to create a machine given a unique key as already discovered
+        # by this plugin.
```

Comment:
We need to implement this TODO, or the whole thing doesn't work, right?

Reply:
I could've sworn it _was_ at one point (by someone else I think). Let me check the merges.

Reply:
Ah, I see, (new work on) `WIP_onboarding_by_ip` was merged into the machine-action branch, not yet into this one.

Reply:
This is implemented in the add_by_ip branch and the machine action branch.